ViewGeneratorCore/tools/package.json

```diff
@@ -8,7 +8,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@outsystems/ts2lang": "1.0.21"
+    "@outsystems/ts2lang": "^1.1.1"
```

Comment:
I think it might be better to have the version fixed here ..

Reply:
ts2lang is using semver, so it should be fine to get newer versions as long as they're not majors (breaking changes)
version locking for the purpose of CI systems should be taken care of by `package-lock.json`